src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -153,6 +150,51 @@ public void getTagsToExcludeFromChart_ReturnsEmptyList() {
         assertThat(patterns).isEmpty();
     }
 
+    @Test
+    public void getDirectorySuffix_ReturnsDirectorySuffix() {
+
+        // given
+        String directorySuffix = "test";
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+
+        // when
+        configuration.setDirectorySuffix(directorySuffix);
+
+        // then
+        assertThat(configuration.getDirectorySuffix()).isEqualTo(directorySuffix);
+    }
+
+    @Test
+    public void getDirectorySuffixWithSeparator_ReturnsDirectorySuffixWithSeparator() {
+
+        // given
+        String directorySuffix = "test";
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+
+        // when
+        configuration.setDirectorySuffix(directorySuffix);
+
+        // then
+        Class<Configuration> configClass = Configuration.class;
+        try {
+            Field suffixSeparator = configClass.getDeclaredField("SUFFIX_SEPARATOR");
+            suffixSeparator.setAccessible(true);
+            assertThat(configuration.getDirectorySuffixWithSeparator()).isEqualTo(suffixSeparator.get(null) + directorySuffix);
+        } catch (NoSuchFieldException|IllegalAccessException ex) {
```

Comment:
still not changed

Reply:
Argh! Another one "commit-no push".  Done now.